packages/conf-libelf/conf-libelf.1.0.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+authors: ["Bayge <alex@fluidity.money>"]
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+build: ["pkg-config" "--exists" "libelf"]
+depexts: [
+  ["libelf-dev"] {os-family = "debian"}
+  ["libelf"] {os-distribution = "arch"}
+  ["dev-libs/libelf"] {os-distribution = "gentoo"}
+  ["libelf"] {os = "freebsd"}
+  ["libelf"] {os = "macos" & os-distribution = "homebrew"}
+  ["libelf-devel"] {os = "win32" & os-distribution = "cygwinports"}
+]
+synopsis: "Virtual package relying on libelf"
+description:
+  "This package can only install if libelf is installed on the system."
```

Comment:
```suggestion
  "This package can only install if libelf is installed on the system."
  depends: ["conf-pkg-config" {build}]
```
The package test depends on `pkg-config`, so this should state a dependency on it.
This is the reason for CI failures `# [ERROR] Command not found: pkg-config`.